### DISCUSSION TITLE
Resurvey interval  for surveillance cameras?

### DISCUSSION THREAD

**Discussion by ERYpTION** [streetcomplete/StreetComplete#4931] [Ideas]
Is there a resurvey interval for man_made=surveillance? If not I would like to suggest adding it. Maybe every 4 years?

**Comment by Helium314** [comment]
It's checked for cameras, see https://github.com/streetcomplete/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/quests/existence/CheckExistence.kt
